src/styles/blocks/card.scss

```
@@ -0,0 +1,115 @@
+@import '../utils/variables';
+
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  margin: 0;
+  height: 406px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+  transition: transform 0.3 ease;
+
+&:hover {
+  transform: scale(1.2);
+}
+
+&__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+  display: block;
+  margin-bottom: 40px;
+}
+
+&__title {
+  color: $main-accent;
+  font-family: inherit;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 35.5px 0 4.4px;
```

Use only integer values
```suggestion
  margin: 35px 0 4px;
```